pkg/manifests/common.go

```diff
@@ -52,14 +52,14 @@ func GetOwnerRefs(owner client.Object, controller bool) []metav1.OwnerReference
 	}}
 }
 
-func Namespace(conf *config.Config) *corev1.Namespace {
+func Namespace(conf *config.Config, nsName string) *corev1.Namespace {
```

Comment:
what is the point of this?

Reply:
decouples creation of a namespace object from the namespace specified in the conf. This'll be necessary when an overlay ns is specified in the externaldns crd

Reply:
We shouldn't infer namespace name from the config anyway - there's nothing inherently about the need to create a namespace object that implies that it must be the one specified in the conf.

If you'd like I can make a second "namespace" method for only externalDNS resources to use (and not nginx ingress controller pods) but that seems like overkill.